app/services/order_details/param_updater.rb

```diff
@@ -84,12 +86,19 @@ def update_attributes(params)
   end
 
   def trigger_notifications
-    OrderDetails::DisputeResolvedNotifier.new(@order_detail,current_user: @editing_user).notify
+    OrderDetails::DisputeResolvedNotifier.new(@order_detail, current_user: @editing_user).notify
     OrderDetails::AssignmentNotifier.new(@order_detail).notify
   end
 
   private
 
+  def log_and_rollback
+    # TODO: Determine why reservations are failing to save at this point, then remove logging.
+    ActiveSupport::Notifications.instrument("background_error", exception: e, information: "Failed save for reservation #{@order_detail.reservation.id}: #{@order_detail.reservation.errors.full_messages}")
```

Comment:
Oh, we don't have an exception here. The undefined `e` is causing test failures.

Also, we should definitely do some sort of test to verify that `information` is ending up in rollbar.

Reply:
I looked at this and decided to stick with just logging to rollbar directly.  The `information` key is used by `ExceptionNotifier`, not Rollbar.